server/app.py

```diff
@@ -209,7 +209,6 @@ async def submit_batch(batch: BatchData):
             f"Size of batch in bytes received for job_id:{batch.job_id} batch_size:{batch_size}"
         )
     except UnknownTopicOrPartitionError:
-        await producer.stop()
```

Comment:
why removed?

Reply:
It's redundant, the `finally` block will execute even if the error is reraised